index.js

```diff
@@ -0,0 +1,127 @@
+    name: 'Multi-Post Stories',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+  {
+    image: '../images/Card 2.png',
+    imageHover: '../images/backround image.png',
+    name: 'Multi-Post Stories',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: ['html', 'bootstrap', 'Ruby'],
+
+  },
+];
+window.onload = () => {
+  const menu = document.querySelector('.hamburger-menu');
+  const close = document.querySelector('.close-icon');
+  const workLink = document.querySelector('.work-link');
+  const aboutLink = document.querySelector('.about-link');
+  const contactLink = document.querySelector('.contact');
+  menu.onclick = () => {
+    document.querySelector('.mobile-menu').classList.remove('hide-menu');
+    document.querySelector('.mobile-menu').classList.add('show-menu');
+  };
+  close.onclick = () => {
+    document.querySelector('.mobile-menu').classList.add('hide-menu');
+  };
+  workLink.onclick = () => {
+    document.querySelector('.mobile-menu').classList.add('hide-menu');
+  };
+  aboutLink.onclick = () => {
+    document.querySelector('.mobile-menu').classList.add('hide-menu');
+  };
+  contactLink.onclick = () => {
+    document.querySelector('.mobile-menu').classList.add('hide-menu');
+  };
+
+  const card = (data) => `<div class="alt-card" style=background-repeat:no-repeat;background-size:cover;">
+    <h2>${data.name}</h2>
+    <p>${data.description}</p>
+    <ul>
+      <li>${data.technologies[0]}</li>
+      <li>${data.technologies[1]}</li>
+      <li>${data.technologies[2]}</li>
+    </ul>
+    <button class="see-project">See Project</button>
+  </div>`;
+
+  const container = document.querySelector('.middle-part');
+  container.innerHTML = datas.map((data) => card(data)).join('');
+  const altCards = document.querySelectorAll('.alt-card');
+
+  const button = document.querySelector('.see-project');
+  altCards.forEach((altCard) => {
+    altCard.onmouseover = () => {
+      button.style.opacity = '1';
+    };
+
+    altCard.onmouseout = () => {
+      button.style.opacity = '0';
+    };
+  });
+
+  const popup = document.querySelector('#popup');
+  const openPopup = () => {
+    popup.style.display = 'block';
+  };
+
+  container.addEventListener('click', (event) => {
+    if (event.target.matches('.see-project')) {
+      openPopup();
+    }
+  });
+
+  button.onclick = () => {
+    popup.style.display = 'block';
+  };
+  const popclose = document.querySelector('.popclose');
+  popclose.onclick = () => {
+    popup.style.display = 'none';
+  };
+};
```

Comment:
- [ ] kindly consider making sure that (on both screen size)  When the user clicks (or taps) the close (X) button, the popup disappears.to have a full functioning popup as requested in the requirements
- and again consider fixing the below popup on mobile screen to have a good implementation of the given design  chek the image below to understand that well 
- your design 👇 
<img width="377" alt="Screen Shot 2023-04-13 at 23 39 31" src="https://user-images.githubusercontent.com/60146030/231889394-27fb7dc4-c5a7-446c-9645-6a1a3f1529b4.png">

- provided design 👇 
<img width="311" alt="Screen Shot 2023-04-13 at 23 43 39" src="https://user-images.githubusercontent.com/60146030/231889711-1bd1d9fd-3ed6-4319-81d1-a56718f3f0f7.png">